src/main/webapp/app/exercises/text/assess/textblock-feedback-editor/textblock-feedback-editor.component.ts

```diff
@@ -177,6 +198,14 @@ export class TextblockFeedbackEditorComponent implements AfterViewInit {
         }
     }
 
+    updateAssessmentWithDropdown(instruction: GradingInstruction) {
+        this.feedback.gradingInstruction = instruction;
+        this.feedback.credits = instruction.credits;
+
+        this.feedback.correctionStatus = undefined;
```

Comment:
Why is this needed?

Reply:
This concerns the tutor training when they assess example submissions and receive feedback on what they have done wrong. The correction status (which can be correct or erroneous) must then be reset. But I'll add a comment to this.